opnode/l1/source.go

```diff
@@ -13,6 +14,11 @@ import (
 	"github.com/ethereum/go-ethereum/ethclient"
 )
 
+var WrongChainErr = errors.New("wrong chain")
+var TooDeepReorgErr = errors.New("reorg is too deep")
+var MaxReorgDepth = 500
+var MaxBlocksInL1Range = uint64(100)
```

Comment:
These can be constants

Reply:
Yep, plus I removed some that were duplicated